frontend/src/components/ExperiencesList.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+    <h2>EXPERIENCE</h2>
```

Comment:
this comment you can resolve in a separate pull request, to not make this very big pull request any bigger

so we have several files (educations, experience, skills, summary) with a structure:
```
title
line
some components
```

the styles are the same, cause I see it was extracted to style.css

We can take it even further, and extract another reusable component. It'd be very helpful if you were to decide that you want more sections with a similar structure.

So, we can extract a wrapper component:
- accepts title
- displays title, line
- contains styles for those elements
- wraps the other content

<img width="650" alt="Screenshot 2024-01-23 at 12 28 52" src="https://github.com/philippOkie/resume-builder-vue/assets/48657633/0e8ff027-c8a6-4967-af67-dc4aaddc4b9e">

red - presentational wrapper component
blue - wrapped content

so you can reuse this wrapper component in views, to make your code more readable and reusable

I recommend reading this https://vuejs.org/guide/components/slots.html to use it for the wrapper component